deployment/chainloop/values.yaml

```diff
@@ -124,6 +124,12 @@ controlplane:
     enabled: false
     allowedOrgs: []
 
+  ## @param controlplane.onboarding[0].name Name of the organization to onboard
+  ## @param controlplane.onboarding[0].role Role of the organization to onboard
+  onboarding:
+    - name: "read-only-demo"
```

Comment:
oh, this shouldn't be here

Reply:
That was the example no?

Reply:
this is not an example, these are the default values. Rendering it without any override will add this onboarding 

```diff
       cas_robot_account_private_key_path: "/secrets/cas.private.key"
@@ -479,6 +479,10 @@
     referrer_shared_index:
       allowedOrgs: []
       enabled: false
+    
+    onboarding:
+      - name: read-only-demo
+        role: MEMBERSHIP_ROLE_ORG_VIEWER

```

Reply:
And yes, I'll comment out onboarding and allowList in values.yaml.

Then the render should show no difference compared to a previous version of the chart